src/test/java/calculator/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,86 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+class StringAddCalculatorTest {
+
+    @ParameterizedTest
+    @NullAndEmptySource
+    @DisplayName("빈 문자열 또는 null 값을 입력하는 경우 0을 반환한다.")
+    void test01(final String text) {
+        // when
+        final var result = StringAddCalculator.splitAndSum(text);
+
+        // then
+        assertThat(result).isZero();
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"1", "2", "3", "4", "5", "6", "7", "8", "9"})
+    @DisplayName("숫자 하나를 문자열로 입력할 경우 해당 숫자를 반환한다.")
+    void test02(final String text) {
+        // when
+        final var result = StringAddCalculator.splitAndSum(text);
+
+        // then
+        assertThat(result).isEqualTo(Integer.parseInt(text));
+    }
```

Comment:
```suggestion
    @ParameterizedTest
    @CsvSource({"1,1", "2,2", "3,3", "4,4", "5,5", "6,6", "7,7", "8,8", "9,9"})
    @DisplayName("숫자 하나를 문자열로 입력할 경우 해당 숫자를 반환한다.")
    void test02(final String text, final int expected) {
        // when
        final var result = StringAddCalculator.splitAndSum(text);

        // then
        assertThat(result).isEqualTo(expected);
    }
```
1. 일반적으로 테스트코드의 검증부에는 가능하면 로직이 들어가지 않는 것이 좋아요. (parseInt와 같은) 물론 지금은 간단한 로직이기는 하지만 하나씩 들어가다보면 테스트코드를 복잡하게하는 원인이 되고 혹시나 그 로직에 잘못된 값을 넣어 원하지 않는 동작을 실행할 수도 있어요.

2. 숫자를 문자열로 넣어 해당 숫자를 반환한다면 9개의 케이스를 돌리기보다는 1개만 돌려도 되지 않을까요? 테스트케이스는 물론 많으면 좋긴하지만 명확하게 테스트하고자하는 엣지케이스만 테스트해도 충분하지 않을까싶어요.